tools/test-proxy/Azure.Sdk.Tools.TestProxy/Azure.Sdk.Tools.TestProxy.csproj

```diff
@@ -1,7 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
-    <TargetFrameworks>net5.0</TargetFrameworks>
-    <RequiredTargetFrameworks>$(TargetFrameworks)</RequiredTargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Comment:
Might be worth adding a comment here of why we are keeping it here and only move it if we move the lower .NET test framework.